files/image_config/hostcfgd/hostcfgd

```diff
@@ -280,6 +242,65 @@ class HostConfigDaemon:
         self.iptables = Iptables()
         self.iptables.load(lpbk_table)
 
+    def update_feature_state(self, feature_name, state, feature_table):
+        has_timer = ast.literal_eval(feature_table[feature_name]['has_timer'])
+        has_global_scope = ast.literal_eval(feature_table[feature_name]['has_global_scope'])
+        has_per_asic_scope = ast.literal_eval(feature_table[feature_name]['has_per_asic_scope'])
+
+        # Create feature name suffix depending feature is running in host or namespace or in both 
+        feature_name_suffix_list = (([feature_name] if has_global_scope or not device_info.is_multi_npu() else []) +
+                                   ([(feature_name + '@' + str(asic_inst)) for asic_inst in range(device_info.get_num_npus()) 
+                                    if has_per_asic_scope and device_info.is_multi_npu()]))
```

Comment:
can we make is_multi_npu() a class member?

Reply:
@tahmed-dev Done